bokehjs/src/lib/models/glyphs/step.ts

```diff
@@ -83,6 +95,45 @@ export class StepView extends XYGlyphView {
     ctx.stroke()
   }
 
+  protected _hit_point(geometry: PointGeometry): Selection {
+    /* Check if the point geometry hits this line glyph and return an object
+    that describes the hit result:
+      Args:
+        * geometry (object): object with the following keys
+          * sx (float): screen x coordinate of the point
+          * sy (float): screen y coordinate of the point
+          * type (str): type of geometry (in this case it's a point)
+      Output:
+        Object with the following keys:
+          * 0d (bool): whether the point hits the glyph or not
+          * 1d (array(int)): array with the indices hit by the point
```

Comment:
This is redundant. Types should explain arguments and products of this function. If there is anything here that's not covered by types, then it can stay (mostly description).